armsrc/fpgaloader.c

```diff
@@ -115,7 +115,7 @@ void SetupSpi(int mode)
 // Set up the synchronous serial port, with the one set of options that we
 // always use when we are talking to the FPGA. Both RX and TX are enabled.
 //-----------------------------------------------------------------------------
-void FpgaSetupSsc(void)
+void FpgaSetupSsc(uint8_t mode)
```

Comment:
What happens to the SSC peripheral if mayor mode is switched to TX while frame length is 16?

Reply:
SSC would continue to send and receive 16Bit frames. If FPGA provides a TF every 8 bits, every second TF is ignored.